liquibase-standard/src/main/java/liquibase/command/core/HistoryCommandStep.java

```diff
@@ -37,6 +40,13 @@ public class HistoryCommandStep extends AbstractCommandStep {
                 .defaultValue(DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT), "Platform specific 'short' format")
                 .hidden()
                 .build();
+        TAGS_ARG = builder.argument("tags", Boolean.class)
```

Comment:
These two command argument names seem pretty similar. I would rename the boolean one to something like "includeTaggedChangesets" or something like that. I would also add to its description that it will include any tagged changeset regardless of whether a tag filter is provided or not.

Reply:
@MalloD12 could we discuss it in the jira ticket https://datical.atlassian.net/browse/DAT-19013 so we keep the specification history there?

Reply:
Sure